scripts/run-operator.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+# simple script to build and run the operator
+cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" || exit
+cd ..
+echo "building operator maven project"
+mvn package --projects :kroxylicious-operator --also-make -DskipTests
+cd kroxylicious-operator || exit
+
+if ! minikube status
+then
+  echo "starting minikube"
+  minikube start
+fi
+
+echo "building operator image in minikube"
+minikube image build . -t quay.io/kroxylicious/operator:latest --build-opt=build-arg=KROXYLICIOUS_VERSION=0.9.0-SNAPSHOT
+
+echo "installing kafka (no-op if already installed)"
+kubectl create namespace kafka
+kubectl create -f 'https://strimzi.io/install/latest?namespace=kafka' -n kafka
+kubectl wait -n kafka deployment/strimzi-cluster-operator --for=condition=Available=True --timeout=300s
+kubectl apply -f https://strimzi.io/examples/latest/kafka/kraft/kafka-single-node.yaml -n kafka 
+kubectl wait kafka/my-cluster --for=condition=Ready --timeout=300s -n kafka
```

Comment:
Can we be consistent about the placement of the `-n kafka` (preferably immediately after the verb option)?